app/models/job_proxy_dispatcher.rb

```diff
@@ -93,6 +94,10 @@ def dispatch
     _log.info "Complete - Timings: #{t.inspect}"
   end
 
+  def container_image_classes
+    ContainerImage.descendants.append(ContainerImage)
```

Comment:
1. We can check for jobs with target != VmOrTemplate.
It is a hack, maybe if we add a comment that states that if we ever get a target that isn't one of the two we should rethink.
2. Maybe we can look at job.type instead